### ISSUE TITLE
make install of init-scripts breaks ability to cross-compile

### ISSUE THREAD

**Issue by lost-carrier** [mikebrady/shairport-sync#66]
I'm trying to cross-compile shairport-sync for Synology NASes (see here: https://github.com/SynoCommunity/spksrc/pull/1617).

I managed to get it compiled, but "make install" fails because ...
Makefile.am:
```
install-exec-hook:
	[ -f /etc/init.d/shairport-sync ] || cp scripts/shairport-sync /etc/init.d/
	update-rc.d shairport-sync defaults 90 10
```
...is PREFIX agnostic and breaks in cross-compiling environment. Not sure what might be the best way to fix it, but something like this might help:
```
install-exec-hook:
        [ -f $(PREFIX)/etc/init.d/shairport-sync ] || cp scripts/shairport-sync $(PREFIX)/etc/init.d/
        if hash update-rc.d 2>/dev/null; then pdate-rc.d shairport-sync defaults 90 10; fi
```

(...since i most probably will have to write my own start-stop-stuff, in theory some "--without-init-d" switch would do it...)

**Comment by mikebrady** [comment]
Hi. Many thanks for looking into this. 

Could I point you at the approach taken by the Arch Linux crowd; they seem to chop out the offending stanza before doing the configuration and make. See the thread of the [closed] pull request at https://github.com/mikebrady/shairport-sync/pull/26 for some references. Would that work for you?

Another thing is that @HiFiJ has been working in this area – see https://github.com/mikebrady/shairport-sync/issues/1. S/he has some problems with synchronisation, etc., but it might be worth comparing notes.

Overall, it would be great to get shairport-sync working for Synology and I'd like to help in any way that I can.

**Comment by lost-carrier** [comment]
Thanks for your reply.

"Joost" sounds like a guy from the Netherlands... ;-) But I think his case is different, since he is trying to stream FROM a Synology box AND to a RasPi...

As far as I got it, the Arch Linux crowd has a very similar problem.

The thing is, that we need a properly maintained repo to get shairport-sync into the SynoCommunity Package Center.

I've added a 
```
head -n 39 Makefile.am.org > Makefile.am
```
somewhere to my scripts which solved it for now - not sure if the SynoCommunity guys will like it. It would be handy to have this in a more configurable way like those ./configure options...
